grug/app/src/state.rs

```diff
@@ -6,37 +6,51 @@ use {
     },
 };
 
+pub const GRUG_NAMESPACE_LEN: usize = 7;
+
+const fn grug_namespace(s: &str) -> &str {
+    debug_assert!(
+        s.len() == GRUG_NAMESPACE_LEN,
+        "string length must be exactly 7"
+    );
+    s
+}
```

Comment:
@Rhaki can you add a comment explaining why it is necessary that all namespaces are the same length